index.json

```diff
@@ -58,6 +58,11 @@
       "mocha/api"
     ]
   },
+  "paper_trail": {
+    "requires": [
+      "active_record"
```

Comment:
I think you have to require https://github.com/paper-trail-gem/paper_trail/blob/master/lib/paper_trail/frameworks/active_record.rb instead. So it knows how to load the `active_record` related features and `active_record` itself.